packages/rich-text/src/create.js

```diff
@@ -146,6 +146,10 @@ export function create( {
 	range,
 	__unstableIsEditableTree: isEditableTree,
 } = {} ) {
+	if ( html instanceof String ) {
+		return html.value;
+	}
```

Comment:
if we sub-class `String` then I think we can make these checks far less ambiguous, like `html instanceof RichTextString` without making other checks fail, because `html instanceof String` would still return `true` here.

Reply:
Switched to a class now